core/widget.php

```diff
@@ -172,7 +176,7 @@ public function save($required_details_form, $widget_options_form, $position_opt
 			$result = DB::exec("update widgets set state=?, title=?, note=?, options=?, position_control=?, global_position=?, page_list=? where id=?", $params);
 			
 			if ($result) {
-				CMS::Instance()->queue_message('Widget updated','success',Config::uripath() . '/admin/widgets/show');	
+				CMS::Instance()->queue_message('Widget <a href="' . Config::uripath() . '/admin/widgets/edit/'.  $this->id . '">' . $this->title .'</a> updated','success', $redirect_url);	
```

Comment:
templating

Reply:
fixed in b772c9972b1b6528cd219293d837976ffad29f09